funk_svd/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .svd import SVD
 from . import utils
 
-__version__ = '0.0.1.dev1'
+__version__ = '0.0.1.dev2'

```

Comment:
I just moved the lib version to a new file `funk_svd/__version__.py`, could you can move the change there?

Reply:
actually I'll just let you manage the version the way you want.